### PR TITLE
fix: Fixing missing import of 'express' in rolesRoutes

### PR DESCRIPTION
**Pull request by bahati10**
### What does this PR do?
Fix Missing import from express
### Description of Task to be completed?
The missing package of express is now added to make sure 
the features at rolesRoutter will work as they expected to be
### How should this be manually tested?

- Create `Role`, `Permission`, and `RolePermission` models.
- Update the `User` model to include a `roleId` field.
- Implement functionality to add and remove permissions from roles.
- Document API endpoints with Swagger.

### CODE REVIEW

**Review by kirengamartial — APPROVED** [review]
this was necessary and it is ready for merge

**Review by Ishimwe7 — APPROVED** [review]
Nice to fix that error

**Review by robsdagreat — APPROVED** [review]
We're good to go @bahati10

File: src/routes/roleRoutes.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-// import { isRoleAdmin } from '../middleware/checkAdminRoleMiddleware.js';
```
Comment (Balinda21) [inline src/routes/roleRoutes.ts:1]: Delete this comment

**Review by Balinda21 — APPROVED** [review]
Well done. This fix was needed!

**Pull request merged**